dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -274,23 +275,50 @@ CWPKI2059I=CWPKI2059I: Certificate revocation status checking found that the cer
+
+CWPKI2066E=CWPKI2066E: The ACME service failed to automatically renew the certificate with the {0} serial number. The certificate is revoked. The request is scheduled to try again in {1}. The renew request error is ''{2}''.
+CWPKI2066E.explanation=The ACME service tried to renew a certificate but encountered an error. The ACME service continues to request a new certificate until a new certificate is issued. SSL/TLS requests fail until a new certificate request is successful.
+CWPKI2066E.useraction=Review the error message for details on the failure. 
+
+CWPKI2067I=CWPKI2067I: The certificate with the {0} serial number is revoked. The ACME service requests a new certificate from the ACME certificate authority at the {1} URI.
+CWPKI2067I.explanation=When the ACME service detects that the certificate is revoked, it automatically requests a new certificate.
+CWPKI2067I.useraction=No action is required.
+
+CWPKI2068W=CWPKI2068W: The ACME service automatic certificate checking failed to check if the certificate with the {0} serial number is expiring or revoked. The check is scheduled to try again in {1}. The error is ''{2}''.
+CWPKI2068W.explanation=The ACME service started checking if the certificate is expiring or revoked, but failed.
+CWPKI2068W.useraction=Review the error message for details on the failure. Review the certificate status using the ACME REST interface. If the certificate needs to be renewed, use the ACME REST interface to request a new certificate.
+
+CWPKI2069I=CWPKI2069I: The ACME service automatic certificate checking is disabled. Expiring or revoked certificates are not automatically renewed.
+CWPKI2069I.explanation=The ACME service does not check for expiring or revoked certificates on an automated schedule. If the certificate expires or is revoked, TLS/SSL requests cannot complete unless the server is restarted or the REST interface is used to renew the certificate.
+CWPKI2069I.useraction=No action is required. To enable automatic certificate checking, update the certCheckerSchedule property in the server configuration to a value greater than 0.
+
+CWPKI2070W=CWPKI2070W: The certCheckerSchedule property was set to {0} which is shorter than the minimum schedule time. The certCheckerSchedule property is reset to {1}.
+CWPKI2070W.explanation=The value for the certCheckerSchedule property was below the minimum duration to check for expiring or revoked certificates and is reset to the minimum schedule time.
+CWPKI2070W.useraction=To avoid this warning message, set the certCheckerSchedule property in the server configuration to a duration that is longer than the minimum schedule time. To use the default setting, remove the certCheckerSchedule property from the server configuration.
+
+CWPKI2071W=CWPKI2071W: The certCheckerErrorSchedule property was set to {0} which is shorter than the minimum schedule time. The certCheckerErrorSchedule property is reset to {1}.
```

Comment:
add comma for dependent clause

The certCheckerErrorSchedule property was set to {0}, which is shorter than the minimum schedule time.

Reply:
Done